From 6ac650626feb8f806af721c4a024b2d9bb360da5 Mon Sep 17 00:00:00 2001
From: Lei Jitang <leijitang@huawei.com>
Date: Thu, 17 Nov 2016 14:04:59 +0800
Subject: [PATCH 1/2] specs-go: add labels to config.go

Signed-off-by: Lei Jitang <leijitang@huawei.com>
---
 specs-go/v1/config.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/specs-go/v1/config.go b/specs-go/v1/config.go
index d42383622695..894cc6cb70f5 100644
--- a/specs-go/v1/config.go
+++ b/specs-go/v1/config.go
@@ -45,6 +45,9 @@ type ImageConfig struct {
 
 	// WorkingDir sets the current working directory of the entrypoint process in the container.
 	WorkingDir string `json:"WorkingDir,omitempty"`
+
+	// Labels contains arbitrary metadata for the container.
+	Labels map[string]string `json:"labels,omitempty"`
 }
 
 // RootFS describes a layer content addresses
-- 
2.11.0

